ci/k6-ingestion-test.js

```diff
@@ -17,8 +17,8 @@ let eventsIngested = new Gauge('events_ingested')
 
 export let options = {
   thresholds: {
-    http_req_failed: ['rate<0.01'],   // http errors should be less than 1%
-    http_req_duration: ['p(95)<500'], // 95% of requests should be below 500ms
+    http_req_failed: ['rate < 0.01'],   // http errors should be less than 1%
+    http_req_duration: ['p(90) < 1000'], // 90% of requests should be below 1s
```

Comment:
Not sure if it's the new PostHog build but latency seems to have increased

Reply:
If this continues, consider making this into an env var to make it elaborate why it's failing there?

Reply:
Looking at the action perf history, this issue seems to happen only on DO where we use very basic droplets with shared resources.